docs/source/contributing/community.rst

```diff
@@ -4,27 +4,19 @@
 Community communication channels
 ================================
 
-We use `Discourse <https://discourse.jupyter.org>` for online discussion.
-Everyone in the Jupyter community is welcome to bring ideas and questions there.
-In addition, we use `Gitter <https://gitter.im>`_ for online, real-time text chat,
-a place for more ephemeral discussions.
-The primary Gitter channel for JupyterHub is `jupyterhub/jupyterhub <https://gitter.im/jupyterhub/jupyterhub>`_.
-Gitter isn't archived or searchable, so we recommend going to discourse first
-to make sure that discussions are most useful and accessible to the community.
-Remember that our community is distributed across the world in various
-timezones, so be patient if you do not get an answer immediately!
+There is a place for everyone in the Jupyter community to share thoughts and ask questions. We use `Discourse <https://discourse.jupyter.org>`_ , `Gitter <https://gitter.im>`_ , and GitHub issues for online discussion.
```

Comment:
This is the first thing people read, so I think it's worth having a concise outline of how we want people to use these. Something like most discussions and support is handled on Discourse. Gitter is for (not sure what we should say..... ), GitHub issues are for discussions related to development, not for user support, see e.g.
https://github.com/jupyterhub/jupyterhub/blob/c126cc3f9bcb9b35d13cc8c942755b1e6575e147/.github/workflows/support-bot.yml#L26

Reply:
Do you think it's better to get rid of line 9 to 19?

Reply:
Not sure, I'm trying to think of some wording that emphasises Discourse as the default place to go to, but mentions the other places as secondary options.

Reply:
Okay. I'll try and come up with something. Or just take inspiration from an already existing doc!